test/SourceKit/ExpressionType/basic.swift

```diff
@@ -31,3 +38,6 @@ func DictS(_ a: [Int: S]) {
 // CHECK: (291, 292): Int?
 // CHECK: (295, 332): Int?
 // CHECK: (295, 320): Int
+// CHECK: (395, 398): String?
+// CHECK: (418, 423): String
+// CHECK: (457, 458): String
```

Comment:
Would you mind adding the other type here? Not sure if it's the `"Hey"`, `"Bye"` or `c` 😅

Reply:
No problem :)